src/compiler/diagnosticMessages.json

```diff
@@ -2560,7 +2565,7 @@
         "category": "Message",
         "code": 6096
     },
-    "File '{0}' exist - use it as a module resolution result.": {
+    "File '{0}' exist - use it as a name resolution result.": {

```

Comment:
answered in [this comment](https://github.com/Microsoft/TypeScript/pull/7775#discussion_r58313500)